components/webhook-mgt/org.wso2.carbon.identity.webhook.management/src/main/java/org/wso2/carbon/identity/webhook/management/internal/dao/impl/CacheBackedWebhookManagementDAO.java

```diff
@@ -166,7 +182,32 @@ public int getWebhooksCount(int tenantId) throws WebhookMgtException {
     @Override
     public List<Webhook> getActiveWebhooks(String eventProfileName, String eventProfileVersion, String channelUri,
                                            int tenantId) throws WebhookMgtException {
-        // Active webhooks retrieval bypasses cache
-        return webhookManagementDAO.getActiveWebhooks(eventProfileName, eventProfileVersion, channelUri, tenantId);
+
+        ActiveWebhooksCacheEntry cacheEntry = ActiveWebhooksCache.getInstance().getValueFromCache(
+                new ActiveWebhooksCacheKey(eventProfileName, eventProfileVersion, channelUri, tenantId), tenantId);
```

Comment:
If we define this (`new ActiveWebhooksCacheKey(eventProfileName, eventProfileVersion, channelUri, tenantId)`) as a local variable, then we can use it in below `addToCache()` method as well

Reply:
Fixed via https://github.com/wso2/carbon-identity-framework/pull/7236/commits/2c5097bc70704062fb00cccaed310669f7540a2b